addons/account/models/account_move.py

```diff
@@ -721,7 +721,7 @@ def _compute_made_sequence_hole(self):
                AND this.sequence_number != 1
                AND this.name != '/'
                AND this.id = ANY(%(move_ids)s)
-               AND (company.fiscalyear_lock_date IS NULL OR this.date >= company.fiscalyear_lock_date)
+               AND (company.fiscalyear_lock_date IS NULL OR this.date > company.fiscalyear_lock_date)
```

Comment:
Regarding the third point of the task, we don't need this line at all.
The filter is only on the dashboard so that the warning doesn't stay until the end of times if there was a mistake in a previous period.
```suggestion
```